docker-compose.yaml

```diff
@@ -26,14 +29,14 @@ services:
       - DATABASE_PASSWORD=postgres
       - DATABASE_NAME=block-explorer
       - BLOCKCHAIN_RPC_URL=http://zksync:3050
+      - BATCHES_PROCESSING_POLLING_INTERVAL=1000
```

Comment:
Decrease the polling interval for batches on local env. Currently the problem is that on the first BatchesProcessor run L1 batches are not yet ready to be processed, so they will be processed only in 60 seconds on the next BatchesProcessor run. As a result, a user won't see batches information for a while when they explore the BE UI. With the decreased timeout batches and all related info appear quickly. I don't think that the frequency of the polling will be an issue on local.